modules/ROOT/pages/ch2-config-endpoints-paths.adoc

```diff
@@ -122,6 +122,25 @@ include::partial$select-private-space.adoc[tag=clickSettings]
 // +
 // <screenshot>
 
+== Sending Application-to-Application Requests within a Private Space
+
+In private space, every application replica exposes an endpoint inside the the private space. When you enable inbound traffic, that endpoint is exposed externally.
```

Comment:
Maybe change to "In a private space,..."

Also, there's an extra "the" in the sentence.

Reply:
```suggestion
In a private space, every application replica exposes an endpoint inside the private space. When you enable inbound traffic, that endpoint is exposed externally.
```